Liquid profile not found for this page

### Problem

I've tried two Shopify stores with a single-login full access account and both times received the message 'This page cannot be profiled'

The console gives me one error:
```
devtools.js:16 Error: Liquid profile not found for this page
```

### Replication steps

I'm not sure but I'm glad to provide more information. I'm possibly not satisfying all of the requirements for this tool

### More Information

The XHR for example.com/collections/all?profile_liquid=true doesn't respond with anything different from a request to example.com/collections/all

--- a/src/devtools.ts
+++ b/src/devtools.ts
@@ -34,10 +34,10 @@ chrome.devtools.inspectedWindow.eval(
 
 function getInspectedWindowURL(): Promise<URL> {
   return new Promise(resolve => {
-    chrome.devtools.inspectedWindow.eval('document.location.href', function(
+    chrome.devtools.inspectedWindow.eval('Shopify.shop', function(
       currentUrl: string,
     ) {
-      resolve(new URL(currentUrl));
+      resolve(new URL(`https://${currentUrl}`));
     });
   });
 }

--- a/src/manifest.json
+++ b/src/manifest.json
@@ -1,6 +1,6 @@
 {
   "name": "Shopify Theme Inspector for Chrome",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Profile and debug Liquid template on your Shopify store",
   "devtools_page": "devtools.html",
   "permissions": ["storage", "identity", "activeTab"],

--- a/test/test-helpers.ts
+++ b/test/test-helpers.ts
@@ -24,8 +24,8 @@ export function setDevtoolsEval(page: any) {
       window.chrome.devtools.inspectedWindow.eval = function(value, cb){
         if (value === "typeof window.Shopify === 'object'") {
           return cb(true)
-        } else if (value === "document.location.href") {
-          return cb('https://shop1.myshopify.io')
+        }  else if (value === "Shopify.shop") {
+          return cb('shop1.myshopify.io')
         }
       };
       window.chrome.devtools.panels.create = () => {};

